framework/include/materials/MaterialData.h

```diff
@@ -128,9 +128,8 @@ class MaterialData
     return haveProperty<T>(prop_name);
   }
 
-  ///@{
   /**
-   * Methods for retieving a MaterialProperty object
+   * @{ Methods for retieving a MaterialProperty object
```

Comment:
What is this doing? Is there some fancy new Doxygen style I don't know about?

Reply:
No, any other option would still work but jut be less efficient.

Reply:
Cool, I have only used the ///@{